src/Models/Model.php

```diff
@@ -26,16 +26,16 @@ abstract class Model implements ModelInterface, Arrayable, JsonSerializable {
 	/**
 	 * The model properties assigned to their types.
 	 *
-	 * @var array<string,string>
+	 * @var array<string,string|array>
 	 */
-	protected $properties = [];
+	static protected $properties = [];
 
 	/**
 	 * The model relationships assigned to their relationship types.
 	 *
 	 * @var array<string,string>
 	 */
-	protected $relationships = [];
+	static protected $relationships = [];
```

Comment:
```suggestion
	protected static $relationships = [];
```

Reply:
Bah! Fixed in [254b408](https://github.com/stellarwp/models/pull/18/commits/254b408d182d184251cacb869292709b809b5193)